### PR TITLE
Labeling fixes

### PR DESCRIPTION
**Pull request by nyalldawson**
Some fixes to allow correct conversion of ESRI style definitions (E.g. https://basemaps.arcgis.com/arcgis/rest/services/World_Basemap_v2/VectorTileServer/resources/styles)

### CODE REVIEW

File: gl2qgis/gl2qgis.py [hunk 1]
```diff
@@ -777,6 +777,8 @@ def parse_symbol_layer(json_layer, style_name):
         text_field = json_layout["text-field"]
         if isinstance(text_field, list):
             label_settings.fieldName = f'\"{text_field[1][1]}\"'
+        else:
+            label_settings.fieldName = str(text_field)
```
Comment (lazaa32) [inline gl2qgis/gl2qgis.py:797]: All labels disapear in case of Basic.

**Comment by lazaa32** [comment]
Hi @nyalldawson 
thanks for contributing and sorry for late answer. I was quite busy and didn't enough time for this plugin. I tested your PR on preloaded maps (Basic, Bright) and with your PR all the labels disappeared. I think it is related to text_field parsing.

**Comment by nyalldawson** [comment]
I'd suggest closing this pr in favour of the new conversion support built into qgis 3.16 - it's got much more robust support for labelling conversion.

**Comment by lazaa32** [comment]
Ok, let's do that. I haven't look at the new classes in 3.16 but it makes sense.

**Comment by nyalldawson** [comment]
https://github.com/maptiler/qgis-maptiler-plugin/issues/133 describes this a little. But the new classes are based off this plugin + many fixes and additions

**Pull request closed without merging**